src/test/scala/edu/ie3/simona/model/thermal/CylindricalThermalStorageSpec.scala

```diff
@@ -41,25 +41,31 @@ class CylindricalThermalStorageSpec
   )
 
   def buildThermalStorage(
-      storageInput: CylindricalStorageInput,
+      cylindricalStorageInput: CylindricalStorageInput,
       volume: Volume,
   ): CylindricalThermalStorage = {
     val storedEnergy = CylindricalThermalStorage.volumeToEnergy(
       volume,
       KilowattHoursPerKelvinCubicMeters(
-        storageInput.getC
+        cylindricalStorageInput.getC
           .to(PowerSystemUnits.KILOWATTHOUR_PER_KELVIN_TIMES_CUBICMETRE)
           .getValue
           .doubleValue
       ),
```

Comment:
Ok I got convinced. With #1358 there will also conversion to squants. Feel free to pick it from there.